AUTHORS.rst

```diff
@@ -62,4 +62,7 @@ Christoph Witzany <christoph at web.crofting.com>
  * Add LDAP support for authentication
 
 Pierre Jaury <pierre at jaury.eu>
-* Docker packaging
\ No newline at end of file
+* Docker packaging
+
+Serban Constantin <serban.constantin at gmail dot com>
+ * Small bug fix
```

Comment:
That file is in general for people who contributed a feature. I would say, that your "Small bug fix" does not count.
If it would it should contain [all contributors](https://github.com/matrix-org/synapse/graphs/contributors).

Reply:
as the CONTRIBUTING.rst file says, anyone making a PR is welcome to add themselves, for any contribution. It's not the size that that matters...

Reply:
Good to know. Then nvm